src/clj/game/core/to_string.clj

```diff
@@ -20,7 +20,6 @@
                          installed-ice " protecting "
                          (is-root? zone) " in the root of "
                          :else " in ")
-                       ;TODO add naming of scoring area of corp/runner
```

Comment:
`name-zone` isn't called in this function lol. The issue is that a stolen agenda will be in `:scored` (with no indication it's runner's score zone), meaning we have to look for the card in both of the score zones and then we can use that to call `name-zone`. Maybe use `find-cid` on both score areas and whichever it's found in, pass that side keyword to `name-zone`.

Reply:
LOL!
Why not replace `zone->name` with `name-zone` in `card-str`?

Reply:
I think cuz the current one pulls the remote number out.